serenity.properties

```diff
@@ -1,5 +1,12 @@
-# Serenity Report properties
-# These properties are used by the Gradle build
-# serenity.project.name = Serenity Test Suite Using JUnit
-# serenity.test.root=starter
-
+serenity.project.name = Serenity_Framework
```

Comment:
please use either serenity.conf or serenity.properties. I recommend the first one